resources/styles/components/exercises.less

```diff
@@ -104,21 +138,27 @@
       line-height: @size;
 
       &.section {
-        border: 1px solid @tutor-neutral-light;
-        min-height: @size;
-        &+.section {
-          border-left: 0;
+        border: 1px solid @default-border;
+        height: @size;
+        position: relative;
+        margin-right: 0;
+        margin-left: -1px;
+
+        &:first-child {
+          margin-left: 0;
         }
       }
       &.active {
         background-color: @tutor-neutral-light;
+        z-index: 10;
       }
```

Comment:
This is because the borders for the buttons are overlapping.  I found that that gave the best effect.  It was the cleanest look.  I could do some really complicated things to change the borders when their siblings are active and stuff, but I thought this was the simplest solution and one that I used for other button groups elsewhere.